server/models/ideaboard.js

```diff
@@ -0,0 +1,47 @@
+const ObjectID = require('mongodb').ObjectID;
+
+const mongoUtil = require('../helpers/mongoUtil.js');
+
+const collectionName = 'ideaboards';
+
+exports.all = (callback) => {
+  mongoUtil.getDb().collection(collectionName).find().toArray((err, result) => {
```

Comment:
It might be a good idea to just project the _id and name field if you're querying every document